vulnerability_id: VCID-7yrn-pn5c-aaac
aliases:
  - CVE-2020-25635
  - PYSEC-2020-220
summary: A flaw was found in Ansible Base when using the aws_ssm connection plugin as garbage
  collector is not happening after playbook run is completed. Files would remain in the bucket
  exposing the data. This issue affects directly data confidentiality.
severities:
  - id: 609884
    reference_id: 931257
    scoring_system: cvssv3
    value: '5.0'
    scoring_elements: CVSS:3.1/AV:L/AC:L/PR:L/UI:R/S:U/C:H/I:N/A:N
  - id: 609883
    reference_id: 931256
    scoring_system: rhbs
    value: medium
    scoring_elements:
  - id: 461934
    reference_id: 22285
    scoring_system: cvssv2
    value: '2.1'
    scoring_elements: AV:L/AC:L/Au:N/C:P/I:N/A:N
  - id: 461932
    reference_id: 22285
    scoring_system: cvssv3
    value: '5.5'
    scoring_elements: CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:H/I:N/A:N
references:
  - id: 931257
    url: https://access.redhat.com/hydra/rest/securitydata/cve/CVE-2020-25635.json
    reference_id:
  - id: 6896
    url: https://bugzilla.redhat.com/show_bug.cgi?id=CVE-2020-25635
    reference_id:
  - id: 6895
    url: https://github.com/ansible-collections/community.aws/issues/222
    reference_id:
  - id: 931256
    url: https://bugzilla.redhat.com/show_bug.cgi?id=1880275
    reference_id: 1880275
  - id: 833999
    url: https://nvd.nist.gov/vuln/search/results?adv_search=true&isCpeNameSearch=true&query=cpe:2.3:a:redhat:ansible:2.10.1:rc2:*:*:*:*:*:*
    reference_id: cpe:2.3:a:redhat:ansible:2.10.1:rc2:*:*:*:*:*:*
  - id: 22285
    url: https://nvd.nist.gov/vuln/detail/CVE-2020-25635
    reference_id: CVE-2020-25635
weaknesses: []
